TrafficCapture/coreUtilities/src/main/java/org/opensearch/migrations/coreutils/MetricsLogBuilder.java

```diff
@@ -0,0 +1,45 @@
+package org.opensearch.migrations.coreutils;
+
+import lombok.extern.slf4j.Slf4j;
+import org.slf4j.event.Level;
+import org.slf4j.Logger;
+import org.slf4j.spi.LoggingEventBuilder;
+
+
+@Slf4j
+public
+class MetricsLogBuilder {
```

Comment:
I'm not sure how to do this, but it would be great to get a semantic model (like SonarQube or IntelliJ) to know that there should be a warning if emit has not been called on these built-up values.  It might already do that by virtue of builders being returned and then discarding the last value.  So - maybe it's a future action item, maybe not

Reply:
Completely agreed--it would be super nice to have.